### DISCUSSION TITLE
How to load other online maps?

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#76] [Q&A]
How to load other online maps?

**Comment by valentine195** [comment]
Can you give an example of what you're trying to do?

**Reply by zhubib1234** [reply]
https://www.bing.com/maps

**Comment by zhubib1234** [comment]
https://www.bing.com/maps

**Reply by valentine195** [reply]
Sorry, this is not going to be possible.

**Reply by zhubib1234** [reply]
well.
